tests/virt_autotest/kubevirt_tests_agent.pm

```diff
@@ -43,15 +46,22 @@ sub rke2_agent_setup {
     my ($self, $server_ip) = @_;
 
     record_info('Start RKE2 agent setup', '');
-    systemctl('stop apparmor.service');
-    systemctl('stop firewalld.service');
+    unless (is_sle_micro) {
+        disable_and_stop_service('apparmor.service');
+        disable_and_stop_service('firewalld.service');
+    }
     $self->setup_passwordless_ssh_login($server_ip);
 
-    # Ensure SUSE certificates are installed on the node
+    # rebootmgr has to be turned off as prerequisity for this to work
+    script_run "rebootmgrctl set-strategy off" if (is_sle_micro);
+    # Check if the package 'ca-certificates-suse' are installed on the node
     ensure_ca_certificates_suse_installed();
+    transactional::process_reboot(trigger => 1) if (is_sle_micro);
```

Comment:
Maybe is_transactional is better, to avoid change for alp in future.

Reply:
Good idea! Already updated.